src/components/PageButtons/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  Button,
+} from '@edx/paragon';
+
+
+export default function PageButtons({prevPage, nextPage, selectedTrack, selectedCohort, getPrevNextGrades}) {
```

Comment:
Instead of taking the arguments as a dictionary like this, what is the challenge of specify each of the dictionary key as argument item?
How difficult would it be to just do 
`PageButtons(prevPage, nextPage, selectedTrack, selectedCohort, getPrevNextGrades)`?

Reply:
Talked about this offline and apparently, this is the better choice to take in parameters on this component. Let's ignore my comment above